app/src/main/java/com/example/kotlin_mvp_calculator/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.kotlin_mvp_calculator.mvp.model
+
+import com.example.kotlin_mvp_calculator.rx.Butns
+
+class CalculatorModel {
+    var firstValue: Double? = null
+        private set
+    var secondValue: Double? = null
+        private set
+    var operator: String? = null
+        private set
+
+    fun reset() {
+        firstValue = null
```

Comment:
why you initialize the values in null? please use a default value to avoid NPE

Reply:
done

Reply:
done